draft-yasskin-wpack-bundled-exchanges.md

```diff
@@ -1,7 +1,7 @@
 ---
 coding: utf-8
 
-title: Bundled HTTP Exchanges
+title: Web Bundles
```

Comment:
We might want to change the file name as well eventually?

Reply:
We might, but it's a bit more difficult to rename IETF drafts. The point at which they get adopted by the working group will be a rename anyway, so we can fix up filenames then, both here and for signed exchanges.